docs/core/diagnostics/built-in-metrics-system-net.md

```diff
@@ -0,0 +1,137 @@
+---
+title: System.Net Metrics
+description: Review the metrics available for System.Net
+ms.topic: reference
+ms.date: 9/21/2023
+---
+
+# System.Net Metrics
+
+This article describes the networking metrics built-in for <xref:System.Net> produced using the
+<xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. For a listing of metrics based on the alternate [EventCounters](event-counters.md) API,
+see [here](available-counters.md).
```

Comment:
>[here](available-counters.md) [](http://example.com/codeflow?start=4&length=29)

Can we add a reference to this metrics page in the available counter page?

Reply:
Added